application/src/main/java/org/togetherjava/tjbot/commands/BotCommandAdapter.java

```diff
@@ -97,6 +98,13 @@ public void onSelectionMenu(SelectMenuInteractionEvent event, List<String> args)
         // Adapter does not react by default, subclasses may change this behavior
     }
 
+    @SuppressWarnings("NoopMethodInAbstractClass")
+    @Override
+    public void onModal(final ModalInteractionEvent event, final List<String> args) {
```

Comment:
the `final` reads a bit weird, being right next to all the "similar methods" who dont have it. id remove here for consistency. or add it everywhere

Reply:
fixed